frontend.java

```diff
@@ -0,0 +1,540 @@
+							//Does there have to be an account name?
+							//make sure they are both valid accounts
+								
+							accountAmount = addZeros(accountAmount, 8);//adds zeros in front if size is less than 8
+							fromAccountNumber = addZeros(fromAccountNumber, 6);//adds zeros in front if size is less than 6
+							toAccountNumber = addZeros(toAccountNumber, 6);//adds zeros in front if size is less than 6
+							
+							
+							String str = "03_" + toAccountNumber + "_" + fromAccountNumber + "_" + accountAmount; 
+							frontend.transactionFile.add(str);
+							//write.writeToSummaryFile(str);
+							System.out.println("Successful Transfer.");
+						}
+					}
+				}
+			}
+		}
+	}
+	public static boolean accountName(String name){
+		//name
+		//Make sure the whole name is letters.
+		boolean valid = true;
+		for (char ch : name.toCharArray()){
+	        int asciiVal = (int)ch;	
+			//if((65 <= asciiVal <= 90) || (97 <= asciiVal <= 122) || (asciiVal == 32)){
+			if((asciiVal >= 65) || (asciiVal <= 90) || (asciiVal >= 97) || (asciiVal <=122)){
+				// a - z
+				// space
+				// A - Z
+				valid=true;
+			}else{
+				valid=false;
+			}
+	    }
+		return valid;
+	}
+	public static boolean accountNumber(String number){
+		//covert the string to integers
+		boolean valid = true;
+		try{
+			int x =Integer.parseInt(number);
+		}catch(NumberFormatException e){
+			valid = false;
+		}
+		//check length make sure the account is the proper length
+		int lengthOfAccount = number.length();
+		if(lengthOfAccount != 8){
+			if(lengthOfAccount > 8){
+				valid = false;
+			}
+		}
+		return valid;
+	}
+	public static boolean accountExists(String number){
+		//Check Account File for the account.
+		
+		//Check length, if its less or equal to 8. 
+		//if less add zeros.
+		if(number.length() == 8){
+			BufferedReader br = null;
+			try {
+	 
+				String sCurrentLine;
+	 
+				br = new BufferedReader(new FileReader("./src/valid_accounts.txt"));
+	 
+				while ((sCurrentLine = br.readLine()) != null) {
+					if(sCurrentLine == number){
+						return true;
+					}
+				}
+	 
+			} catch (IOException e) {
+				e.printStackTrace();
+			}
+		}else if(number.length() > 8){
+			return false;
+		}else{
+			String newAccountNumber = addZeros(number, 8);
+			//check the newAccountNumber
+			BufferedReader br = null;
+			try {
+	 
+				String sCurrentLine;
+	 
+				br = new BufferedReader(new FileReader("./src/valid_accounts.txt"));
+	 
+				while ((sCurrentLine = br.readLine()) != null) {
+					if(sCurrentLine == newAccountNumber){
+						return true;
+					}
+				}
+	 
+			} catch (IOException e) {
+				e.printStackTrace();
+			}
+		}
+		return false;
+	}
+	public static boolean accountAmount(String number){
+		try{
+			int x =Integer.parseInt(number);
+		}catch(NumberFormatException e){
+			return false;
+		}
+		return true;
+	}
+	public static String addZeros(String number, int size){
+		//get the number and subtrack it by 8 and add that many zeros
```

Comment:
This can be replaced with String.format(), which would do what you need to do, and should be familiar to most Java / C / C++ programmer:

belam@belam-amd ~ $ cat foo.java
public class foo {
    public static void main(String[] args){
        String s = String.format("%06d", 123);
        System.out.println(s);
        return;
    }
}
belam@belam-amd ~ $ java foo
000123